features/environment.py

```diff
@@ -23,7 +54,8 @@ def browser_init(context):
 
 def before_scenario(context, scenario):
     print('\nStarted scenario: ', scenario.name)
-    browser_init(context)
+    # Pass scenario.name to init() for browserstack config:
+    browser_init(context, scenario.name)
```

Comment:
Pass `scenario.name` from here to init() if you want to use it in the browserstack config for `'sessionName': scenario_name` (see line 42 above)